espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Handlers/Finder.java

```diff
@@ -108,23 +105,22 @@ private ViewInteraction findBy(Strategy strategy, String selector) throws Server
                     break;
                 case TEXT:
                     // with text
-                    // TODO: Should we have different strategies for 'withText' and 'withContentDescription'?
                     matcher = onView(withText(selector));
-                    if(matcher == null) {
-                        // if text not find, check content description
-                        matcher = onView(withContentDescription(selector));
-                    }
                     break;
-                default:
+                case CONTENT_DESCRIPTION:
+                    // if text not find, check content description
```

Comment:
the previous _break_ should be removed if one follows the logic declared by this comment